src/typography/__tests__/Code.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import faker from 'faker'
+import Code from '../src/Code'
+
+describe('Props', () => {
+  test('<Code /> accepts arbitrary props', () => {
```

Comment:
Thanks for the tests! A couple of small nits for consistency's sake: 

- Do you mind changing these to `describe` and `it` blocks? 
- The passthrough of arbitrary props is actually already tested by [ui-box](https://github.com/segmentio/ui-box). Could we keep the test case scoped to `className` forwarding?
- What do you think about just adding a smoke test that the component renders as expected with a "happy path"?

Reply:
@akleiner2 Thanks for the feedback!

* Removed the arbitrary testing of props
* Added "happy path" tests (using snapshots)
* `test` -> `it`

35e227d, 1d60bf9

Let me know if I'm missing anything else 😃